src/app/cli/src/init/graphql_client.ml

```diff
@@ -1,170 +0,0 @@
-open Core
```

Comment:
Is this just github showing a move as a delete or is this an actual delete as far as git is concerned? Just nice to preserve history/blame when we can